src/main/java/com/odysseusinc/arachne/datanode/controller/BaseDataSourceController.java

```diff
@@ -174,12 +181,15 @@ protected BaseDataSourceController(UserService userService,
                     e.setModelType(dto.getModelType());
                 }
             });
+        } catch (RetryableException e) {
+            LogUtils.logError(LOGGER, COMMUNICATION_FAILED, e);
+            throw new ServiceNotAvailableException("Central is not available");
+        } catch (AuthenticationException e) {
```

Comment:
401 from Central is not treated as auth exception:
![image](https://user-images.githubusercontent.com/10010071/63176076-41102700-c013-11e9-84c1-2ea5663cbed3.png)